### ISSUE TITLE
Jobs are not logged

### ISSUE THREAD

**Issue by felix-d1strict** [D1strict/AL-JobTracker#5]
Sometimes there is an error that there is a JSON error. Which does not fix itself. And therefore does not log any jobs. A message should be output as soon as the expected data is not output.

**Comment by felix-d1strict** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Jobs are not logged](https://trello.com/c/QzYRyhul/16-jobs-are-not-logged)

**Comment by AdyStudios** [comment]
Do you know, when does it happens?

**Comment by felix-d1strict** [comment]
Unfortunately not. I can't see any pattern there.

**Comment by AdyStudios** [comment]
I will do some tests then, I hope there is a pattern.

**Comment by felix-d1strict** [comment]
I'll have a bit more time next week, so I'll see if I can reproduce it. And whether I can get the error message.

**Comment by felix-d1strict** [comment]
There is no pattern. The error occurs at irregular intervals. Sometimes after starting a job, sometimes half an hour after starting the job, sometimes when submitting the job. It is strange and makes it harder to find the error. Possibly it's also an ETCars error. idk.

**Comment by AdyStudios** [comment]
We could try to maybe get an older version of etcars (might sound stupid but I will try it)

**Comment by felix-d1strict** [comment]
Unfortunately I don't have the exact error message anymore, but it may have been this one: https://github.com/shardick/etcars-node-client/issues/3

So the problem is known.

**Comment by felix-d1strict** [comment]
Since I don't get an error message, I'll assume that the error has been fixed.

**Comment by AdyStudios** [comment]
Hmm, something I noticed is that the tracker has to be running before ets2 is started (you may already know this)

**Comment by felix-d1strict** [comment]
**There are API errors which come from etcars.**

After long conversations with a developer of etcars, I have now decided that we will move away from etcars. Not in the next weeks and months - but in the future.

So far we could not agree on a good solution regarding etcars - or its successor "Truckler".
On the one hand "Trucker" is quite low-maintenance, because it is delivered ready to use.
On the other hand, I miss the individualization. Unfortunately, we can't implement any additional functions there because, as I said, the software is delivered ready-made.

Also, the developer does not want to provide the backend, which I can understand.

I have now found an alternative and will soon (in a few weeks or months (as I still have some customer orders)) if the alternative works & if we can use it.
Then I will rewrite our JobTracker. But that in the distant future.
